tests/arp/test_wr_arp.py

```diff
@@ -190,6 +191,31 @@ def setupRouteToPtfhostFixture(self, duthosts, rand_one_dut_hostname, ptfhost):
         yield
         self.teardownRouteToPtfhost(duthost, route, ptfIp, gwIp)
 
+    @pytest.fixture(scope='class', autouse=True)
+    def warmRebootSystemFlag(self, duthost):
+        """
+            Sets warm-reboot system flag to false after test. This class-scope fixture runs once before test start
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+
+            Returns:
+                None
+        """
+        yield
+        if not wait_until(300, 10, 0, self.checkWarmbootFlag, duthost):
+            logger.info('Setting warm-reboot system flag to false')
+            duthost.shell(cmd='sonic-db-cli STATE_DB hset "WARM_RESTART_ENABLE_TABLE|system" enable false')
+
+    def checkWarmbootFlag(self, duthost):
+        """
+            Checks if warm-reboot system flag is set to false.
+        """
+        warmbootFlag = duthost.shell(
+            cmd='sonic-db-cli STATE_DB hget "WARM_RESTART_ENABLE_TABLE|system" enable')['stdout']
+        logger.info("warmbootFlag: " + warmbootFlag)
+        return warmbootFlag == 'false'
```

Comment:
You should check the flag not equal to 'true'. The flag could be empty after cold reboot.

Reply:
addressed